nbtc/relayer.go

```diff
@@ -139,6 +170,35 @@ func (r *Relayer) processSignedTxs() error {
 	return nil
 }
 
+// fetchAndStoreSignRequests fetches and stores sign requests from the Native chain.
+func (r *Relayer) fetchAndStoreNativeSignRequests() error {
+	//TODO: decide how many sign requests we want to process at a time
+	signRequests, err := r.signReqFetcher.GetBtcSignRequests(r.signReqStart, 5)
```

Comment:
We should set the number of sign requests as an env variable.

Reply:
maybe part of the config when initalizing?

Reply:
Yes :D This is better than pushing this to env in the case.

Reply:
Done

Reply:
here, config better than env ;)

Reply:
Ah. Yesterday, I think the config better. But I don't know why I write env here 😭